src/util/tools.ts

```diff
@@ -18,3 +19,14 @@ export const isContract = async (provider: Provider, address: string): Promise<b
 export const range = (max: number): number[] => {
   return Array.from(new Array(max), (_, i) => i)
 }
+
+export const trivialPartition = (size: number) => {
```

Comment:
Can you add an small description to this method? What is the purpose? what is it for?